test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -200,6 +200,25 @@ public void execute_list_showsAllPersons() throws Exception {
                               expectedList);
     }
 
+
+    @Test
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.